include/net/if.h

```diff
@@ -44,6 +44,20 @@
 #define IF_NAMESIZE        16  /* Newer naming standard */
 #define IFHWADDRLEN        6
 
+#ifdef CONFIG_NET_OFFLOAD
+/* Interface feature bits */
+
+#define NETIF_F_UDP_GSO    (1 << 0)  /* Interface supports the UDP GSO */
```

Comment:
```suggestion
#define NETIF_F_GSO_UDP    (1 << 0)  /* Interface supports the GSO UDP  */
```
move all NIC feature define to netdev_features.h and align feature name with linux:
https://github.com/torvalds/linux/blob/master/include/linux/netdev_features.h#L150

Reply:
Move the definitions to netdev.h, and rename the macros.

Reply:
Done